tests/postgres/worker.test.ts

```diff
@@ -22,7 +22,7 @@ jest.mock('../../src/worker/ingestion/ingest-event')
 jest.mock('../../src/worker/plugins/run')
 jest.mock('../../src/worker/plugins/setup')
 jest.mock('../../src/worker/plugins/teardown')
-jest.setTimeout(600000) // 600 sec timeout
+jest.setTimeout(1000000) // 1000 sec timeout
```

Comment:
why the change?

Reply:
So I definitely was annoying here: https://github.com/PostHog/plugin-server/pull/314/commits/5626c5dea5ea4197b5e2b5a648f6e621fe8436cb

Sorry for not reverting that - but the worker benchmarks were failing on that PR. I guess the timeout is somewhat part of the benchmark too? The thing is that it was throwing a rather cryptic error